editor/state.js

```diff
@@ -136,6 +156,26 @@ export const blocks = combineUndoableReducers( {
 }, { resetTypes: [ 'RESET_BLOCKS' ] } );
 
 /**
+ * Reducer returning the last-known state of the current post, in the format
+ * returned by the WP REST API.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Dispatched action
+ * @return {Object}        Updated state
+ */
+export function currentPost( state = {}, action ) {
```

Comment:
Shouldn't most of the UI around current status, title, author, categories, tags, etc. be based on this `currentPost` state key?